include/aspect/particle/property/crystal_preferred_orientation.h

```diff
@@ -37,12 +37,12 @@ namespace aspect
     {
       enum class DeformationType
       {
-        passive
+        passive, olivine_a_fabric, olivine_b_fabric, olivine_c_fabric, olivine_d_fabric, olivine_e_fabric, enstatite
```

Comment:
I know that we don't have documentation in the other places here either, but it would be nice to have short doxygen documentation what each of the enums means.